unit_tests/test_sct_events_health.py

```diff
@@ -27,35 +28,34 @@ def test_cluster_health_validator_event(self):
         self.assertTrue(issubclass(ClusterHealthValidatorEvent.NodeSchemaVersion, ClusterHealthValidatorEvent))
         self.assertTrue(hasattr(ClusterHealthValidatorEvent, "NodesNemesis"))
         self.assertTrue(issubclass(ClusterHealthValidatorEvent.NodesNemesis, ClusterHealthValidatorEvent))
-        self.assertTrue(hasattr(ClusterHealthValidatorEvent, "ClusterHealthCheck"))
-        self.assertTrue(issubclass(ClusterHealthValidatorEvent.ClusterHealthCheck, ClusterHealthValidatorEvent))
```

Comment:
Please, add checks for `Info` and `Done` subevents.

Reply:
done